packages/scandipwa/src/route/StyleGuidePage/StyleGuidePage.component.tsx

```diff
@@ -53,6 +54,25 @@ export class StyleGuidePageComponent extends PureComponent<StyleGuidePageCompone
         [ ADDITIONAL_ELEMENTS ]: (): ReactElement => this.renderAdditionalElements(),
     };
 
+    tabMap = [
+        {
+            id: 'First tab',
+            name: __('First tab'),
+            render: (): ReactElement => 'First content',
+        },
+        {
+            id: 'Second tab',
+            name: __('Second tab'),
+            shouldTabBeRemoved: () => true,
+            render: (): ReactElement => 'Second content',
+        },
+        {
+            id: 'Third tab',
+            name: __('Third tab'),
+            render: (): ReactElement => 'Third content',
+        },
+    ];
```

Comment:
This turned out to be very hard to extend, maybe we can instead redo it in methods?